README.md

```diff
@@ -2,105 +2,145 @@
     </picture>
 </p>
 
 [![Join the chat at https://discord.gg/f6aerfE](https://img.shields.io/discord/500285081265635328.svg?style=flat&logo=discord&label=discord&logoColor=f2f2f2)](https://discord.gg/f6aerfE)
 [![All Contributors](https://img.shields.io/github/all-contributors/stride3d/stride?color=ee8449)](#contributors)
 [![Financial sponsors](https://img.shields.io/opencollective/all/stride3d?logo=opencollective)](https://opencollective.com/stride3d)
 [![License](https://img.shields.io/badge/license-MIT-blue)](https://github.com/stride3d/stride/blob/master/LICENSE.md)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](CODE_OF_CONDUCT.md)
 
-Welcome to the Stride source code repository!
+# Welcome to the Stride Game Engine
 
-Stride is an open-source C# game engine for realistic rendering and VR. 
-The engine is highly modular and aims at giving game makers more flexibility in their development.
-Stride comes with an editor that allows you to create and manage the content of your games or applications visually and intuitively.
+Stride is an open-source C# game engine designed for realistic rendering and VR. Highly modular, it aims to give game makers more flexibility in their development. Stride comes with an editor, [Game Studio](https://doc.stride3d.net/latest/en/manual/game-studio/index.html), which allows you to create and manage the content of your games or applications visually and intuitively. To learn more about Stride, visit [stride3d.net](https://stride3d.net/).
 
 ![Stride Editor](https://stride3d.net/images/external/script-editor.png)
 
-To learn more about Stride, visit [stride3d.net](https://stride3d.net/).
+## Table of Contents
 
-## Earn money by contributing
-If you are a developer with solid experience in C#, rendering techniques, or game development, we want to hire you! We have allocated funds from supporters on [OpenCollective](https://opencollective.com/stride3d) and can pay for work on certain projects. [More info about this here](https://doc.stride3d.net/latest/en/contributors/engine/bug-bounties.html).
+* 🚀 [Getting Started](#-getting-started)
+* 🤝 [Contributing](#-contributing)
+  - [Earn Money by Contributing](#earn-money-by-contributing)
+* 🗺️ [Roadmap](#%EF%B8%8F-roadmap)
+* 🛠️ [Building from Source](#%EF%B8%8F-building-from-source)
+  - [Prerequisites](#prerequisites)
+  - [Build Stride](#build-stride)
+  - [Build Stride without Visual Studio](#build-stride-without-visual-studio)
+  - [If Building Fails](#if-building-fails)
+  - [Contribution Guidelines](#contribution-guidelines)
+* 🔬 [Build Status](#-build-status)
+* 📖 [Stride Documentation Landscape](#-stride-documentation-landscape)
+* 🌐 [.NET Foundation](#-net-foundation)
+* 🛡️ [License](#%EF%B8%8Flicense)
+* ✨ [Contributors](#-contributors)
```

Comment:
Looks like github have added a TOC-like feature:
![image](https://github.com/user-attachments/assets/84a597f9-640e-451e-8e14-a5fa7cbc84de)

Reply:
They should name it as `TOC` or something so it is easier to discover 🤣. Anyway, that's awesome. I can remove my suggested TOC, including from other repos because it is not irrelevant. Thanks for suggesting this!

Reply:
Done.